packages/dom/src/serialize-video.js

```diff
@@ -12,7 +12,11 @@ export function serializeVideos(dom, clone) {
 
     canvas.getContext('2d').drawImage(video, 0, 0, width, height);
 
-    let dataUrl = canvas.toDataURL();
+    let dataUrl;
+    try {
+      dataUrl = canvas.toDataURL();
+    } catch (e) {}
```

Comment:
We can drop the `(e)`

```suggestion
    } catch {}
```